src/triton_object_recognition/config/yolov3.yaml

```diff
@@ -0,0 +1,98 @@
+#There is an issue with ComposableNode where it doesn't understand the typical nested YAML structure
+#https://github.com/ros2/rclcpp/issues/715
+
+model_folder: "src/triton_object_recognition/models/"
```

Comment:
A better way to do this would be to always have the models directory, and add it to the package exactly like how you add a launch directory, so that we can use the ament_index method get_package_share_directory to get the models dir. (https://github.com/ament/ament_index/blob/47236e623aa93e596457a1fa18741a302f7ff967/ament_index_cpp/src/get_package_share_directory.cpp)